src/mumble/mumble.pro

```diff
@@ -386,6 +386,16 @@ win32 {
   win32-msvc* {
     LIBS *= -lsndfile -lvorbis -lvorbisfile -logg -lFLAC
   }
+
+  # Boost
+  LIBS *= -lboost_system-mt
```

Comment:
Why are we including this for every platform?

Reply:
This is in the win32 block.

Reply:
Ah, sorry. I didn't see it because I didn't expand the diff.